Include/internal/pycore_code.h

```diff
@@ -30,6 +30,11 @@ typedef struct {
     uint32_t builtin_keys_version;
 } _PyLoadGlobalCache;
 
+typedef union {
+    _PyAttrCache attr;
+    PyObject *meth;  /* borrowed */
+} _PyLoadMethodCache;
+
```

Comment:
Make this a struct containing the `PyObject *` and maybe name it as `PyObjectCache`.
This will be the third cache for `LOAD_MERHOD` after `_PyAdaptiveEntry` and `_PyAttrCache`.